taichi/backends/vulkan/snode_struct_compiler.cpp

```diff
@@ -79,7 +79,7 @@ int SNodeDescriptor::cells_per_container_pot() const {
   if (ty == SNodeType::root || ty == SNodeType::place) {
```

Comment:
ditto

Reply:
For this one I mean Line 77 & root check in Line 79 :-) GitHub doesn't show that line I guess